.circleci/config.yml

```diff
@@ -6,6 +6,8 @@ executors:
       - image: circleci/golang:1.13
 
 jobs:
+# GitHub Actions equivalent of this job (.github/workflows/client.yml)
```

Comment:
As test-go is the only workflow we've got configured in Circle CI maybe we could remove this configuration file at all.

Reply:
You're right. I think we can even delete whole `.circleci` folder - thats's what I just commited.